src/Microsoft.ML.Core/Environment/HostEnvironmentBase.cs

```diff
@@ -460,41 +452,6 @@ protected virtual IProgressChannel StartProgressChannelCore(HostBase host, strin
             return new ProgressReporting.ProgressChannel(this, ProgressTracker, name);
         }
 
-        /// <summary>
-        /// This "opens" the input file handle. The default implementation assumes the file resides in the
-        /// file system and it does not access the physical file until a read stream is opened on the returned
-        /// file handle.
-        /// </summary>
-        protected virtual IFileHandle OpenInputFileCore(IHostEnvironment env, string path)
-        {
-            this.AssertValue(env);
-            this.CheckNonWhiteSpace(path, nameof(path));
```

Comment:
Are we missing these checks in the new Utility methods?

Reply:
TODO: 
- [ ] bring back checks to utility methods